ninja-core/src/main/java/ninja/utils/NinjaConstant.java

```diff
@@ -127,6 +127,11 @@
      */
     final String serverName = "application.server.name";
 
+    /**
+     * Whether to display logo at startup. Defaults to true if missing.
+     */
+    final String NINJA_SPLASH_DISPLAY = "ninja.splash.display";
```

Comment:
Very nice! One thing we need is documentation - otherwise nobody would know about your great feature... I think configuration_and_modes.md would be a good candidate to add some lines regarding ninja.splash.display and ninja/logo.txt